scalafix-tests/unit/src/test/scala/scalafix/tests/util/IntervalSetSpec.scala

```diff
@@ -0,0 +1,46 @@
+package scalafix.tests.util
+
+import org.scalatest.FunSuite
+import org.scalactic.source.Position
+
+import scalafix.internal.util.IntervalSet
+
+class IntervalSetSpec() extends FunSuite {
+  // format: off
```

Comment:
Not convinced the minor vertical alignment is paying off here, can we please skip this? You can make the comments align by adding more spaces. I personally once bailed out on a PR to a project I was contributing to just because there was a lot of manual vertical alignment and I didn't know how to keep things pretty. It's a distraction.